client/src/views/page-components/statistics-page/RecentlyAccountHistory/RecentlyAccountHistory.ts

```diff
@@ -0,0 +1,68 @@
+import { PICTOGRAM } from '@src/static/constants';
+import { AccountData } from '@src/types';
+import { changeIntoDateFormat, createDOMWithSelector } from '@src/utils/helper';
+
+import './RecentlyAccountHistory.scss';
+
+export default class RecentlyAccountHistory {
+  $RecentlyAccountHistory: HTMLElement;
+  recentlyAccountHistoryList: AccountData[];
+
+  constructor({ parent, state }) {
+    this.$RecentlyAccountHistory = createDOMWithSelector('div', '.recently-account-history');
+    this.recentlyAccountHistoryList = state;
+
+    parent.appendChild(this.$RecentlyAccountHistory);
+    this.render();
+  }
+
+  render() {
+    this.$RecentlyAccountHistory.innerHTML = `
+        <span class='recently-account-history__title'>최근 가계부 내역</span>
+        <a href='/account' class='recently-account-history__detail'>자세히 보기 ></a>
+        <div class='recently-account-wrap'>
+          ${this.getRecentlyAccountsDOM(this.recentlyAccountHistoryList)}
+        </div>
+    `;
+  }
+
+  /**
+   * 최근 가계부 내역에 해당하는 DOM들을 반환합니다.
+   */
+  getRecentlyAccountsDOM = (data: AccountData[]): string => data.map((d) => this.RecentlyAccount(d)).join('');
+
+  /**
+   * RecentlyAccount 컴포넌트입니다.
+   * 추후에 components로 이동할 가능성이 있지만,
+   * 협업자와의 합의가 필요하다고 생각해 해당 위치에 선언했습니다.
+   */
+  RecentlyAccount(data: AccountData): string {
```

Comment:
이런식으로 위와 같은 이유로 바꿔주시면  좋을거 같습니다!
```suggestion
  type HTMLtext = string;
  RecentlyAccount(data: AccountData): HTMLtext  { 
```

Reply:
아주 좋네요!!